src/Raven.Server/Documents/Schemas/Attachments.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Sparrow.Server;
+using Voron;
+using Voron.Data.Tables;
+
+namespace Raven.Server.Documents.Schemas
+{
+    public static class Attachments
+    {
+        public static TableSchema Current => AttachmentsSchemaBase60;
+
+        internal static readonly TableSchema AttachmentsSchemaBase = new TableSchema();
+        internal static readonly TableSchema AttachmentsSchemaBase60 = new TableSchema();
+
+        internal static readonly Slice AttachmentsSlice;
+        internal static readonly Slice AttachmentsMetadataSlice;
+        internal static readonly Slice AttachmentsEtagSlice;
+        internal static readonly Slice AttachmentsHashSlice;
+        internal static readonly Slice AttachmentsTombstonesSlice;
+        internal static readonly Slice AttachmentsBucketAndEtagSlice;
+        internal static readonly string AttachmentsTombstones = "Attachments.Tombstones";
+
+        internal enum AttachmentsTable
+        {
+            /* AND is a record separator.
+             * We are you using the record separator in order to avoid loading another files that has the same key prefix,
+                e.g. fitz(record-separator)profile.png and fitz0(record-separator)profile.png, without the record separator we would have to load also fitz0 and filter it. */
+            LowerDocumentIdAndLowerNameAndTypeAndHashAndContentType = 0,
+            Etag = 1,
+            Name = 2, // format of lazy string key is detailed in GetLowerIdSliceAndStorageKey
+            ContentType = 3, // format of lazy string key is detailed in GetLowerIdSliceAndStorageKey
+            Hash = 4, // base64 hash
+            TransactionMarker = 5,
+            ChangeVector = 6
+        }
+
+        static Attachments()
+        {
+            using (StorageEnvironment.GetStaticContext(out var ctx))
+            {
+                Slice.From(ctx, "Attachments", ByteStringType.Immutable, out AttachmentsSlice);
+                Slice.From(ctx, "AttachmentsMetadata", ByteStringType.Immutable, out AttachmentsMetadataSlice);
+                Slice.From(ctx, "AttachmentsEtag", ByteStringType.Immutable, out AttachmentsEtagSlice);
+                Slice.From(ctx, "AttachmentsHash", ByteStringType.Immutable, out AttachmentsHashSlice);
+                Slice.From(ctx, "AttachmentsBucketAndEtag", ByteStringType.Immutable, out AttachmentsBucketAndEtagSlice);
+                Slice.From(ctx, AttachmentsTombstones, ByteStringType.Immutable, out AttachmentsTombstonesSlice);
+            }
+
+            DefineIndexesForAttachmentsSchema(AttachmentsSchemaBase);
+            DefineIndexesForAttachmentsSchemaBase60();
```

Comment:
I like this approach, it will ensure that we have proper way to track changes over time